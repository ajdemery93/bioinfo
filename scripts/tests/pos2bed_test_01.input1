chr1	10
chr1	19
chr1	24
