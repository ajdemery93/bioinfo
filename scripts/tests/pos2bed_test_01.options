 --posfile pos2bed_test_01.input1 --faifile pos2bed_test_01.input2
