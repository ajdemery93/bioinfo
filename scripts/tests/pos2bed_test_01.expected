chr1	0	9
chr1	9	18
chr1	18	23
chr1	23	31
